crates/ironrdp-svc/src/lib.rs

```diff
@@ -62,6 +58,79 @@ pub trait StaticVirtualChannel: AsAny + fmt::Debug + Send + Sync {
 
 assert_obj_safe!(StaticVirtualChannel);
 
+/// Takes a vector of PDUs and breaks them into chunks prefixed with a [`ChannelPDUHeader`].
+/// Each chunk is at most `max_chunk_len` bytes long (not including the ChannelPduHeader).
+pub fn chunkify(pdus: Vec<Box<dyn PduEncode>>, max_chunk_len: usize) -> PduResult<Vec<WriteBuf>> {
```

Comment:
Just thinking out loud: I could see this function elsewhere as it’s not going to be used by implementers of static channels, but it’s not something we have to think about right now.